projects/go-lib/src/lib/components/go-copy/go-copy.component.scss

```diff
@@ -1,10 +1,14 @@
+.go-copy {
+  position: relative;
+}
+
 .go-copy__textarea {
-  position: absolute;
   left: 99999px;
+  position: absolute;
 }
 
-.go-copy {
-  overflow: hidden;
-  position: relative;
+.go-copy__content {
```

Comment:
Is there an html element that has this class on it?

Reply:
Yeah, I forgot to update that so good catch. I added this class to the appropriate element in the template file.